qa/tasks/caasp.py

```diff
@@ -0,0 +1,100 @@
+'''
+Task that deploys a CAASP cluster on all the nodes
+Linter:
+    flake8 --max-line-length=100
+'''
+import logging
+from teuthology.misc import (
+    all_roles_of_type,
+    append_lines_to_file,
+    get_file
+    )
+from teuthology.task import Task
+from util import (
+    get_remote_for_role,
+    )
+log = logging.getLogger(__name__)
+
+
+class Caasp(Task):
+    """
+    Deploy a Caasp cluster on all remotes using Skuba.
+    Nodes are declared in suites/caasp folder
+    """
+
+    def __init__(self, ctx, config):
+        super(Caasp, self).__init__(ctx, config)
+        log.debug("beginning of constructor method")
+        self.ctx['roles'] = self.ctx.config['roles']
+        self.log = log
+        self.remotes = self.cluster.remotes
+        self.mgmt_remote = get_remote_for_role(self.ctx, "skuba_mgmt_host.0")
```

Comment:
Maybe just rename 'self.mgmt_remote' to 'self.mgmt' since it is clear that all nodes are remote.

Reply:
Done